docs/using.rst

```diff
@@ -460,13 +460,19 @@ repo, if you have not already done so. Then run:
 
 .. code-block:: shell
 
-   sudo apt-get install letsencrypt python-letsencrypt-apache -t jessie-backports
+   sudo apt-get install certbot python-certbot-apache -t jessie-backports
 
 **Fedora**
 
 .. code-block:: shell
 
-    sudo dnf install letsencrypt
+    sudo dnf install certbot
+
+if you want to use the Apache plugin:
+
+.. code-block:: shell
+   
+   sudo dnf install python-certbot-apache
```

Comment:
This appears to be called `python2-certbot-apache`. I created certbot/website#155 to fix the issue on the website.